lra/coordinator/server/pom.xml

```diff
@@ -36,9 +36,6 @@
     <description>Narayana compatible LRA coordinator</description>
 
     <properties>
-        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
I'm concerned that this module is not deployable because its parent is `io.helidon.applications:helidon-se` and therefore won't have all the stuff needed to be deployable (javadocs, source jar, signed jars, etc).

Reply:
fixed